content/knowledge-base/guides/composition-revisions.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Composition Revisions
+---
```

Comment:
Dropping a note that this is a beta feature and should be tagged clearly as such.

Reply:
added in a unique PR. https://github.com/crossplane/docs/pull/263